src/muya/lib/index.js

```diff
@@ -119,7 +122,9 @@ class Muya {
     this.contentState.importMarkdown(newMarkdown)
     this.contentState.importCursor(cursor)
     this.contentState.render(isRenderCursor)
-    this.dispatchChange()
+    setTimeout(() => {
+      this.dispatchChange()
+    }, 0)
```

Comment:
Do you found this issue, the `word count` is zero every time when we open a new markdown file?

Because the `change` event listener is registered after muya init, but muya `setMarkdown` immediately, so the `change` listener can not know the first change event.